packages/tailwindcss/src/compile.ts

```diff
@@ -321,3 +322,38 @@ function getPropertySort(nodes: AstNode[]) {
 
   return Array.from(propertySort).sort((a, z) => a - z)
 }
+
+/**
+ * Sort an array of entries into an array-of-sets. Similar entries (where the
+ * sort function returns 0) are grouped together. The sort function is stable
+ * and uses bubble-sort
```

Comment:
Is the comment about bubble sort useful if we are just using `.sort()` and not implementing the algorithm ourselves?

Also wonder if we should use `toSorted()`/make a copy since this function has a return value — is it weird that it has the side effect of sorting one of the parameters in place?

Reply:
If we're using `sort` then it's *not* bubble sort — it's most likely quick sort but I think the comment about the sort function should just go 🤷 

The only use of this passes in `getUsedVariants(…)` which is a fresh array every time so sorting in place is more efficient.

Reply:
> Is the comment about bubble sort useful if we are just using .sort() and not implementing the algorithm ourselves?

No that's a leftover, mb!

> Also wonder if we should use toSorted()/make a copy since this function has a return value — is it weird that it has the side effect of sorting one of the parameters in place?

Yeah Jordan and me were also thinking. I did have it operate on a clone first but given that this is pretty hot code we thought that the optimization might make sense. That said, since its a function now it feels like something that another callsite might want to use, at which point this will become confusing yeah